src/components/__tests__/ProgramList.test.js

```diff
@@ -111,13 +111,13 @@ describe('The ProgramList component', () => {
         removeProgram={removeProgram}
         fetchTags={fetchTags}
         clearProgram={clearProgram}
-        user={{ user_id: 1 }}
+        user={{ user_id: 1, username: 'testuser' }}
         programs={null}
       />,
     ).dive();
 
     expect(wrapper.find(ProgramCollection).exists()).toBe(true);
-    expect(wrapper.find(ProgramCollection).length).toBe(1);
+    expect(wrapper.find(ProgramCollection).length).toBe(2);
     expect(wrapper.find(Loader).exists()).toBe(true);
```

Comment:
I'm not sure that these tests are really asserting the proper thing. Basically, they each pass if any `Loader` is present. They should be checking that there is a `Loader` in the specific `ProgramCollection` which is `null` for that test

Reply:
Yeah, the were a little tighter when there were only two ProgramCollections.

Reply:
Added a check that there's only one loader. Still not perfectly specific, but a little better.